TSKit.xcodeproj/project.pbxproj

```diff
@@ -646,10 +646,12 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/TSKit/lib",
 				);
+				ONLY_ACTIVE_ARCH = NO;
```

Comment:
It's a common habit to leave the this setting to `YES` for Debug builds to speed the builds during development

Reply:
Ya, Sorry that was me trying to see if I could get it working for my project in both a simulator and an actual device. I currently have to compile it for device and then import it into the project to get it to work but then it freaks out if you try and run on simulator. I will switch it back because it did not have the desired effect.